src/GeohashHilbert.jl

```diff
@@ -261,7 +285,15 @@ function cell_size_deg(precision, bits_per_char = 2)
     return 360 / n, 180 / n
 end
 
-function neighbours(geohash, bits_per_char)
+"""
+    neighbours(geohash, bits_per_char = 2)
+
+Compute the geohashes of the cells neighboring the specified geohash cell. Return a
+dictionary with keys "north", "north-east", "east", etc. and values the geohash string of
```

Comment:
Nit: Maybe more clear imo to say "keyed by" and "valued by", or have a colon following "keys" and "values". Reads fine as is.